common/src/test/java/com/amazonaws/services/schemaregistry/common/AWSSchemaRegistryClientTest.java

```diff
@@ -139,13 +139,6 @@ public void testAWSSchemaRegistryClient_putSchemaVersionMetadata_succeeds() {
         }
     }
 
-    @Test
```

Comment:
This test should be unnecessary - we should trust the default credentials provider.